config/settings.yml

```diff
@@ -16,6 +16,9 @@ accounts:
   product_default: 75340
   revenue_account_default: 50617
 
+admin_holds:
+  repeat_end_date_in_weeks: <%= ENV.fetch("ADMIN_HOLD_REPEAT_END_DATE_IN_WEEKS", 12) %>
```

Comment:
What do you think about documenting this and the other options managed via settings.yml?
We have `HOWTO_feature_flags.md` that describes some of it, but I think we should aim to document more of the settings either in that file, a new file, or perhaps in notion.  What do you think?

Reply:
After talking about this, decided to consider documenting in the future